source/components/home-page/home-page.js

```diff
@@ -39,67 +39,93 @@ class HomePage extends YummyRecipesComponent {
       this.shadowRoot.getElementById("recipe-card-grid-" + i).innerHTML = "";
     }
 
+    // create an array of recipe object
+    let random_recipe = [];
```

Comment:
Please use `const`. Also since this array contains only the recipes and their indices, I'm not sure if using the word `random` works here. Maybe something like the below?
```suggestion
    const recipeObjects = [];
```